README.md

```diff
@@ -52,6 +52,37 @@ typically needed or wanted for a production setup. Require
   `Access-Control-Allow-Headers` in a preflight response
 - `:body`: Clojure data structure to be encoded based on the `:content-type`
 
+## Examples
+
+A simple JSON get request.
+
+``` clojure
+(require '[lambdaisland.fetch :as fetch])
+
+(-> (fetch/get "https://reqres.in/api/users/2")
+    (.then (fn [resp]
+             (-> resp
+                 :body
+                 (js->clj :keywordize-keys true)
```

Comment:
Use of `js->clj` is generally a bad practice, we shouldn't be setting a bad example here.

Reply:
@plexus Thanks, fixed!